### ISSUE TITLE
Support for Lat, Long, and Province/State

### ISSUE THREAD

**Issue by rlindskog** [pomber/covid19#9]
Hey, great project! It's nice having a cached JSON representation of CSSEGISandData/COVID on github!

What do you think about adding lat / long attributes? Also province/states (maybe in a separate JSON file)?

I can help if it is something you're interested in.

**Comment by pomber** [comment]
Yes! do you want to send a PR changing the action to also create/update a `timeline-by-region.json` or something similar?

There's also `countries.json` with some info about each country, feel free to add more info to it.

**Comment by titaniumbones** [comment]
I added provincial breakdown here: https://github.com/titaniumbones/covid19 . I don't understand github actions, therefore can't test them, and wanted to get this up ASAP so I just copied `actions.js` to `manual.js` and made a few changes there. extracted some logic into named functions. More than happy to make a PR but I won't be able to test till I have a chance to learn about actions (maybe tonight but maybe not.

**Comment by titaniumbones** [comment]
(I also added the CSSE repo as a submodule, which makes it easy to update w/ a shell script.)

**Comment by pomber** [comment]
Awesome @titaniumbones. 

I'm not sure if you can test the action locally (I don't have too much experience with actions either)

But you could change action.js and test it in your repo in every pull by changing 
https://github.com/pomber/covid19/blob/a4873cb40130b1aed596bb4e2c39bc9a234c34dc/.github/workflows/main.yml#L1-L3 
to
```yaml
on: [push]
```

If it works, send de PR and I'll merge it.

**Comment by titaniumbones** [comment]
I think I was able to do this in #13 :-/ -- anyway, actions seem to work.  on my repo.  Also possible to test by running `GITHUB_WORKSPACE=somedir` after adding the relevant files into `somedir/data` and `somedir/main`

**Comment by jparish3** [comment]
great work...have provinces been added yet?

**Comment by Jaydenaus** [comment]
Any updates on this? Working on a tracker that I'd really love to be able to support state/provinces as well.

**Comment by pomber** [comment]
There are some new changes coming to the source repo (#37). 
I would need to change how the data is fetched so I'll add provinces then (in a different json).

**Comment by bakman2** [comment]
would be nice if the countries could be mapped to their ISO 2 or 3 code, which in turn can be mapped to lon/lat. There are several nodejs libs available that have all the information required. 
(and perhaps even a proper ISO date format and deaths today property)

**Comment by izaius** [comment]
Would really like state/province level info. Thank you for your work on this so far!

**Comment by energee** [comment]
Long story short, the JHU data is a complete mess, particularly in the daily series there are key issues in multiple places. I was working on pull request for this repo, but required too many hacks to make the api's data consistent. Anyway, I made an API for US state data available here: https://github.com/energee/covid19-us-api based on NY times csv. Hopefully its helpful!

**Comment by jsanjay63** [comment]
Taking the idea from this project, I made available COVID-19 data in GeoJSON format. 
Repo URL: https://github.com/jsanjay63/covid19-geoextractor

Sample Data available: https://jsanjay63.github.io/covid19-geoextractor/output/covid19_time_series.geojson

Map Visualization: https://studio.here.com/viewer/?project_id=fc72e094-6ade-4c2f-8e9c-ecf902b36709

Feel free to give feedback or contribute. A small contribution to fight COVID :)
